apps/webapp/app/routes/_app.orgs.$organizationSlug.projects.v3.$projectParam.deployments/route.tsx

```diff
@@ -120,39 +120,37 @@ export default function Page() {
-                              <div
-                                key={status}
-                                className="grid grid-cols-[8rem_1fr] gap-x-2 py-2 first:pt-1 last:pb-1"
-                              >
-                                <div className="mb-0.5 flex items-center gap-1.5 whitespace-nowrap">
-                                  <DeploymentStatus status={status} isBuilt={false} />
-                                </div>
-                                <Paragraph
-                                  variant="extra-small"
-                                  className="!text-wrap text-text-dimmed"
-                                >
-                                  {deploymentStatusDescription(status)}
-                                </Paragraph>
+                    <TableHeaderCell>Deploy</TableHeaderCell>
+                    <TableHeaderCell>Env</TableHeaderCell>
+                    <TableHeaderCell>Version</TableHeaderCell>
+                    <TableHeaderCell
+                      tooltip={
+                        <div className="flex flex-col divide-y divide-grid-dimmed">
+                          {deploymentStatuses.map((status) => (
+                            <div
+                              key={status}
+                              className="grid grid-cols-[8rem_1fr] gap-x-2 py-2 first:pt-1 last:pb-1"
+                            >
+                              <div className="mb-0.5 flex items-center gap-1.5 whitespace-nowrap">
+                                <DeploymentStatus status={status} isBuilt={false} />
                               </div>
-                            ))}
-                          </div>
-                        }
-                      >
-                        Status
-                      </TableHeaderCell>
-                      <TableHeaderCell>Tasks</TableHeaderCell>
-                      <TableHeaderCell>Deployed at</TableHeaderCell>
-                      <TableHeaderCell>Deployed by</TableHeaderCell>
-                      <TableHeaderCell hiddenLabel>Go to page</TableHeaderCell>
-                    </TableRow>
+                              <Paragraph
+                                variant="extra-small"
+                                className="!text-wrap text-text-dimmed"
+                              >
+                                {deploymentStatusDescription(status)}
+                              </Paragraph>
+                            </div>
+                          ))}
+                        </div>
+                      }
+                    >
+                      Status
+                    </TableHeaderCell>
+                    <TableHeaderCell>Tasks</TableHeaderCell>
+                    <TableHeaderCell>Deployed at</TableHeaderCell>
+                    <TableHeaderCell>Deployed by</TableHeaderCell>
+                    <TableHeaderCell hiddenLabel>Go to page</TableHeaderCell>
```

Comment:
"You must place `<tr>` elements inside `<thead>` to contain your `<th>` cells. The correct structure is: `<thead><tr><th>Header Cell</th></tr></thead>`. [1](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/thead) [2](https://www.w3.org/TR/html401/struct/tables.html)"